spec/menu_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'menu'
+
+describe Menu do
+  describe '#dishes' do
+    it 'returns the MENU constant' do
+      expect(subject.dishes).to eq(Menu::MENU)
+    end
+  end
+  
+  describe '#user_menu' do
+    it 'prints the available dishes and their prices' do
+      expect { subject.user_menu }.to output.to_stdout
```

Comment:
Can we make a stronger assertion here? What if `user_menu` just printed "hello world" to stdout - would this test pass? Should it?